phys/module_ra_rrtmg_sw.F

```diff
@@ -10251,6 +10252,10 @@ SUBROUTINE RRTMG_SWRAD(                                        &
     integer:: idx_rei
     real:: corr
 
+#ifdef CLWRFGHG
+! Using data from CAMtr_volume_mixing_ratio data file
+    REAL(8)                                      :: co2, n2o, ch4, cfc11, cfc12
```

Comment:
@weiwangncar 
Wei,
Here and the identical code in swf:
Replace
```
REAL(8)
```
with
```
REAL(KIND=8)
```

Reply:
@davegill The code in module_ra_rrtmg_lw.F uses real(8). We may need to change that later.

Reply:
@weiwangncar 
Wei,
No probs at all to only do some of this. We can't fix all questionable syntax, but when we come to it, we can clean it up. To me, it is fine to leave the other code AS-IS.